<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WebSpider</class>
 <widget class="QMainWindow" name="WebSpider">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1112</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WebSpider</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="font">
         <font>
          <family>Roboto Condensed</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: 2px solid rgb(28, 255, 47);
background: rgb(22, 22, 22);
color: rgb(32, 194, 14);</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">text-align: center;
</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>150</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Roboto Condensed</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
          <underline>false</underline>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(0, 0, 0);
border: 0px;
color: rgb(32, 194, 14)</string>
        </property>
        <property name="title">
         <string>WebSpider</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="4" colspan="2">
          <widget class="QPushButton" name="buttonStop">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>22</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto Condensed</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(32,194,14);
color: black;</string>
           </property>
           <property name="text">
            <string>Stop</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineUrl">
           <property name="font">
            <font>
             <family>Roboto Condensed</family>
             <pointsize>12</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 0px;
background: black;
color: rgb(32,194,14);
border-bottom: 2px solid rgb(32,194,14);</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4" colspan="2">
          <widget class="QPushButton" name="buttonStart">
           <property name="minimumSize">
            <size>
             <width>120</width>
             <height>22</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto Condensed</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">background: rgb(32,194,14);
color: black;</string>
           </property>
           <property name="text">
            <string>Start</string>
           </property>
          </widget>
         </item>
         <item row="6" column="5">
          <widget class="QSpinBox" name="spinBoxThreads">
           <property name="font">
            <font>
             <family>Roboto Condensed</family>
             <pointsize>12</pointsize>
             <weight>50</weight>
             <italic>true</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 0px;
background: black;
color: rgb(32,194,14);
border-bottom: 2px solid rgb(32,194,14);
</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="labelSearch">
           <property name="font">
            <font>
             <family>Roboto Condensed</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(32,194,14);</string>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineSearch">
           <property name="font">
            <font>
             <family>Roboto Condensed</family>
             <pointsize>12</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 0px;
background: black;
color: rgb(32,194,14);
border-bottom: 2px solid rgb(32,194,14);</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="lineLimit">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Roboto Condensed</family>
             <pointsize>12</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">border: 0px;
background: black;
color: rgb(32,194,14);
border-bottom: 2px solid rgb(32,194,14);</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelUrl">
           <property name="font">
            <font>
             <family>Roboto Condensed</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(32,194,14);
font-family: &quot;Roboto Condensed&quot;;
</string>
           </property>
           <property name="text">
            <string>URL</string>
           </property>
          </widget>
         </item>
         <item row="6" column="4">
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <family>Roboto Condensed</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(32,194,14);
text-align: left;</string>
           </property>
           <property name="text">
            <string>Threads</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Roboto Condensed</family>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
             <underline>false</underline>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(32,194,14);</string>
           </property>
           <property name="text">
            <string>Limit</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>60</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1112</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>lineUrl</tabstop>
  <tabstop>lineSearch</tabstop>
  <tabstop>lineLimit</tabstop>
  <tabstop>textBrowser</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
